routes/productRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router  = express.Router();
+
+const productController = require('../controllers/productController');
+
+router.get('/detail', productController.getProductInfoByProductId);
```

Comment:
특정 `product`의 상세 정보를 처리하는 방법으로 `path parameters`를 사용할 수 있을 것 같네요!
나머지 부분도 path parameter를 사용하여 특정 데이터 정보에 접근할 수 있을 것 같습니다.

이에 따른 controller 단에서의 코드도 수정해주세요~!

```suggestion
router.get('/:productId', productController.getProductInfoByProductId);
```